insights/client/connection.py

```diff
@@ -802,6 +804,25 @@ def register(self):
         else:
             return (message, client_hostname, "None", "")
 
+    def _archive_too_big(self, archive_file):
+        '''
+        Some helpful messaging for when the archive is too large for ingress
+        '''
+        archive_filesize = size_in_mb(
+            os.stat(archive_file).st_size)
+        logger.info("Archive is {fsize} MB which is larger than the maximum allowed size of {flimit} MB.".format(
+            fsize=archive_filesize, flimit="100"))
+
+        if not self.config.core_collect:
+            logger.error("Cannot estimate the spec with largest filesize because core collection is not enabled. "
+                    "Enable core collection by setting core_collect=True in %s, and attempt the upload again.", self.config.conf)
+            return
+
+        biggest_file = largest_spec_in_archive(archive_file)
+        if biggest_file is not None:
+            logger.info("The largest file in the archive is %s at %s MB.", biggest_file[0], size_in_mb(biggest_file[1]))
+            logger.info("Please add this spec:\n\n\t%s\n\nto your denylist configuration according to the documentation, and try the upload again.", biggest_file[2])
```

Comment:
Good question. I don't know what documentation it's referring to. Maybe we can point the user to the file path rather than the ambiguous "the documentation".